src/backends/onnxruntime.c

```diff
@@ -0,0 +1,217 @@
+#include "backends/onnxruntime.h"
+#include "tensor.h"
+#include "util/alloc.h"
+#include "util/arr_rm_alloc.h"
+
+#if 0
+TF_DataType RAI_GetTFDataTypeFromDL(DLDataType dtype) {
+
+  if (dtype.code == kDLFloat) {
+    switch (dtype.bits) {
+      case 32:
+        return TF_FLOAT; break;
+      case 64:
+        return TF_DOUBLE; break;
+      default:
+        return 0;
+    }
+  }
+  else if (dtype.code == kDLInt) {
+    switch (dtype.bits) {
+      case 8:
+        return TF_INT8; break;
+      case 16:
+        return TF_INT16; break;
+      case 32:
+        return TF_INT32; break;
+      case 64:
+        return TF_INT64; break;
+      default:
+        return 0;
+    }
+  }
+  else if (dtype.code == kDLUInt) {
+    switch (dtype.bits) {
+      case 8:
+        return TF_UINT8; break;
+      case 16:
+        return TF_UINT16; break;
+      default:
+        return 0;
+    }
+  }
+  return 0;
```

Comment:
Possible to remove return.

Reply:
I'm not sure why here. While it's true that `DLDataTypeCode` can only be `kDLInt`, `kDLInt`, `kDLInt`, the `DLDataType` struct has `code` as `uint8_t` (see https://github.com/dmlc/dlpack/blob/master/include/dlpack/dlpack.h#L99), so we can't exclude there's an unexpected value there.

Would you rather `abort`?